generic/src/merge.rs

```diff
@@ -60,7 +60,15 @@ impl Numberer {
     }
 }
 
+macro_rules! blah {
+    ($($args:tt),*) => {
+        let _span = tracing::debug_span!($($args),*).entered();
+    };
+}
```

Comment:
We should put this somewhere but I'm not sure where. `prjunnamed_netlist`? New `prjunnamed_log` crate?